### PR TITLE
removed ctrl.Log

### PR DESCRIPTION
**Pull request by mikebz**
old techdebt (b/264312480) to remove the ctrl.Log

### CODE REVIEW

File: cmd/reconciler-manager/main.go [hunk 1]
```diff
@@ -92,10 +93,10 @@ func main() {
 		setupLog.Error(err, "failed to build dynamic client")
 		os.Exit(1)
 	}
-	watchFleetMembership := fleetMembershipCRDExists(dynamicClient, mgr.GetRESTMapper())
+	watchFleetMembership := fleetMembershipCRDExists(dynamicClient, mgr.GetRESTMapper(), &setupLog)
```
Comment (tiffanny29631) [inline cmd/reconciler-manager/main.go:96]: Adding an argument here is not necessary as the function can use the global var declared on L49

Comment (mikebz) [reply cmd/reconciler-manager/main.go:96]: I believe L49 is declared within main, I have removed the setupLog from globals.  I don't think you can just grab variables up the stack, but maybe I am wrong?

Comment (tiffanny29631) [reply cmd/reconciler-manager/main.go:96]: Yeah you are right, overlooked the removal.

**Review by tiffanny29631 — APPROVED** [review]
/lgtm

**Pull request merged**